rse-cmdline-git.Rmd

```diff
@@ -0,0 +1,3536 @@
+# Git at the command line {#cmdline-git}
+
+## Questions {#cmdline-git-questions}
+
+```{r, child="questions/cmdline-git.md"}
+```
+
+## Objectives {#cmdline-git-objectives}
+
+```{r, child="objectives/cmdline-git.md"}
+```
+
+## Introduction {#cmdline-git-intro}
+
+A version control system stores a master copy of your code in a repository,
+which you can’t edit directly.
+Instead, you checkout a working copy of the code,
+edit that code, then commit changes back to the repository.
```

Comment:
Define "commit". Could add these to the glossary.

What's the target audience here? Beginner or intermediate?